### DISCUSSION TITLE
Choose Rust For...

### DISCUSSION THREAD

**Discussion by dbanty** [dbanty/dylananthony.com#24] [Ideas]
The general idea here is a "series" (it's not really serial, more like a bunch of related posts) that deep dives into the many reasons I love Rust and want to switch to it for most if not all of development. This should go beyond cool Rust features into the more practical aspects of using it.

#23 would likely be the first of these posts, showing why the Rust language and community have provided for a better way of interacting with databases than I've seen in Python.

Like the general idea of the series? Drop a 👍 or upvote this thread (since that seems to be a discussions feature now). Have an idea for a specific post which falls under this category? Pop it below in a comment.

**Comment by imbolc** [comment]
Hey, an obvious next step would be covering frontent frameworks I guess :)

**Reply by dbanty** [reply]
I suppose so, though I'm not much of a front end developer myself. For sure I'll experiment with them at some point though I'm not sure I'm qualified to declare them "better" than something like React.

I suppose at the very least I could talk about some of the pains of JavaScript in the browser and how Rust might alleviate them.
